cfgov/jinja2/v1/_includes/molecules/breadcrumbs.html

```diff
@@ -0,0 +1,27 @@
+{# ==========================================================================
+
+   breadcrumbs.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create breadcrumb markup when given:
+
+   items:              An array of tuples used to display nav items.
+                       format: (href, id, caption)
+
+   additional_classes: Extra classes you wish to add to the nav,
+                       space separated.
+
+   ========================================================================== #}
+
+{% macro render(breadcrumbs, additional_classes) %}
```

Comment:
Fine for now, but we really need to remove `additional_classes` from atomic elements. If arbitrary classes can be added it's not encapsulated—it's not atomic. This needs to be handled through BEM modifiers, boolean flags passed into the render method, or by defining a separate molecule. Can you add a `TODO`?